class-wc-facebookcommerce.php

```diff
@@ -96,6 +96,8 @@ public function init() {
 				require_once $this->get_framework_path() . '/utilities/class-sv-wp-async-request.php';
 				require_once $this->get_framework_path() . '/utilities/class-sv-wp-background-job-handler.php';
 
+				require_once __DIR__ . '/includes/API/Response.php';
```

Comment:
@jdeeburke I realized we didn't have a story to instantiate the API so I added [one](https://app.clubhouse.io/skyverge/story/55024) and copied Authorize.Net where we require the necessary files in that method right before creating the new instance.

Does that sound good? If it does, I'll make sure move these two to the other method in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1338 now that this one is merged.

Reply:
@wvega Yeah actually I was thinking this felt like the wrong spot for those requires. That's a good move 👍 Thanks!